config/distscript.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env sh
+#
+# Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
+#                         University Research and Technology
+#                         Corporation.  All rights reserved.
+# Copyright (c) 2004-2005 The University of Tennessee and The University
+#                         of Tennessee Research Foundation.  All rights
+#                         reserved.
+# Copyright (c) 2004-2005 High Performance Computing Center Stuttgart,
+#                         University of Stuttgart.  All rights reserved.
+# Copyright (c) 2004-2005 The Regents of the University of California.
+#                         All rights reserved.
+# Copyright (c) 2009-2014 Cisco Systems, Inc.  All rights reserved.
+# Copyright (c) 2015      Los Alamos National Security, LLC. All rights
+#                         reserved.
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+srcdir="$1"
+builddir=$PWD
+distdir="$builddir/$2"
+OMPI_VERSION="$3"
+OMPI_REPO_REV="$4"
+
+if test x"$2" = x ; then
+    echo "Must supply relative distdir as argv[2] -- aborting"
+    exit 1
+elif test x"$OMPI_VERSION" = x ; then
+    echo "Must supply version as argv[1] -- aborting"
+    exit 1
+fi
+
+# we can catch some hard (but possible) to do mistakes by looking at
+# our repo's revision, but only if we are in the source tree.
+# Otherwise, use what configure told us, at the cost of allowing one
+# or two corner cases in (but otherwise VPATH builds won't work)
+repo_rev=$OMPI_REPO_REV
+if test -d .git ; then
+    repo_rev=$(config/opal_get_version.sh VERSION --repo-rev)
+fi
+
+start=$(date)
+cat <<EOF
+
+Creating Open MPI distribution
+In directory: `pwd`
+Version: $OMPI_VERSION
+Started: $start
+
+EOF
+
+umask 022
+
+# Some shell startup files alias cp, mv, and rm to have "-i"
+# (interactive).  Unalias here, just so that we don't use that option.
+unalias cp &> /dev/null
+unalias rm &> /dev/null
+unalias mv &> /dev/null
+
+if test ! -d "$distdir" ; then
+    echo "*** ERROR: dist dir does not exist"
+    echo "*** ERROR:   $distdir"
+    exit 1
+fi
+
+#
+# Update VERSION:repo_rev with the best value we have.
+#
+sed -e 's/^repo_rev=.*/repo_rev='$repo_rev'/' "${distdir}/VERSION" > "${distdir}/version.new"
+cp -f "${distdir}/version.new" "${distdir}/VERSION"
+rm -f "${distdir}/version.new"
+# need to reset the timestamp to not annoy AM dependencies
+touch -r "${srcdir}/VERSION" "${distdir}/VERSION"
+echo "*** Updated VERSION file with repo rev number"
+
+#########################################################
+# VERY IMPORTANT: Now go into the new distribution tree #
+#########################################################
+cd "$distdir"
+echo "*** Now in distdir: $distdir"
+
+#
+# Get the latest config.guess and config.sub from ftp.gnu.org
+#
```

Comment:
Where did you get this config.guess/config.sub stuff from?  We don't do this on master any more -- did you start with a v1.8 version of distscript.csh?

None of this code should be on master any more (lines 86 through 167)